### ISSUE TITLE
Allow Robots to be able to set their 'Status' profile data

### ISSUE THREAD

**Issue by GoogleCodeExporter** [hitrust/google-wave-resources#576]
```
I would envisage this as an addon to the profile.xml file.

If settable, this line could be used for informing users of the robot's 
main function. It's not a major leap compared to the rest of wave, but 
sometimes I've entered into a conversation, seen a robot present that I 
don't have a clue what it does and I end up 'fighting' against it. If that 
robot had told me that it removed all blips bar the first one etc, I would 
have had a much better user experience. 

As a developer, I understood what was going on but I'm wondering if an 
actual user would realize that there is a robot doing that behaviour. 
Instead, they may think it is a bug in Wave itself.

In a nutshell, nothing extraordinary but just handy.
```

Original issue reported on code.google.com by `wonderso...@gmail.com` on 10 Dec 2009 at 6:59
